apps/codeforafrica/src/pages/api/admin/config.js

```diff
@@ -930,5 +930,314 @@ module.exports = {
+              fields: [
+                {
+                  label: "Src",
+                  name: "src",
+                  widget: "image",
+                },
+                {
+                  label: "Height",
+                  name: "height",
+                  widget: "string",
+                  required: false,
+                },
+                {
+                  label: "Width",
+                  name: "width",
+                  widget: "string",
+                  required: false,
+                },
+              ],
+            },
+            {
+              label: "Description",
+              name: "description",
+              widget: "text",
+            },
+            {
+              label: "Social",
+              name: "social",
+              widget: "object",
+              fields: [
+                {
+                  label: "Title",
+                  name: "title",
+                  widget: "string",
+                },
+                {
+                  label: "Links",
+                  name: "links",
+                  label_singular: "Link",
+                  widget: "list",
+                  summary: "{{link.content}}",
+                  fields: [
+                    {
+                      label: "Link",
+                      name: "link",
+                      widget: "object",
+                      fields: [
+                        {
+                          label: "Content",
+                          name: "content",
+                          widget: "string",
+                        },
+                        {
+                          label: "Href",
+                          name: "href",
+                          widget: "string",
+                        },
+                        {
+                          label: "Icon",
+                          name: "icon",
+                          widget: "object",
+                          fields: [
+                            {
+                              label: "Src",
+                              name: "src",
+                              widget: "image",
+                            },
+                            {
+                              label: "Height",
+                              name: "height",
+                              widget: "string",
+                              required: false,
+                            },
+                            {
+                              label: "Width",
+                              name: "width",
+                              widget: "string",
+                              required: false,
+                            },
+                          ],
+                        },
+                      ],
+                    },
+                  ],
+                },
+                {
+                  label: "Main Navigation Links",
+                  name: "main-navigation-links",
+                  label_singular: "Main Navigation Link",
```

Comment:
1. Move this outside Social/Stay in touch. It should be a standalone widget under Footer.
2. Can it just link/reference `Main navigation` in Header?
```suggestion
                  label: "Main Navigation",
                  name: "main-navigation",
                  label_singular: "Navigation Link",
```

Reply:
For `Can it just link/reference Main navigation in Header?` unless we make Navigation an individual `Collection` and use `relation` field.